### PR TITLE
Removes tightly coupled font class names

### PR DESCRIPTION
**Pull request by dmitrinesterenko**
[Addresses #48]

### CODE REVIEW

File: pattern-library/src/typography/modules/font-weights.scss [hunk 1]
```diff
@@ -1,19 +1,15 @@
...
 }
 
-.tisa-med,
 .serif-med
 {
   font-family: $font-family-serif;
   font-weight: 500;
 }
 
-.tisa-sans,
```
Comment (GeorgeTaveras1231) [inline pattern-library/src/typography/modules/font-weights.scss]: Looks good but I think the references to these classes need to be removed from `typography.md` file

Comment (dmitrinesterenko) [inline pattern-library/src/typography/modules/font-weights.scss]: @GeorgeTaveras1231 looks that the typography.md file had been removed in an earlier PR. I merged master in here and that file is no longer present.

**Comment by dmitrinesterenko** [comment]
@GeorgeTaveras1231 check this out again please.

File: pattern-library/src/typography/modules/font-weights.scss (hunk 1 above)
Comment (GeorgeTaveras1231) [inline pattern-library/src/typography/modules/font-weights.scss]: The file is [here](https://github.com/xogroup/union/blob/master/pattern-library/docs/articles/pattern-library/core-components/typography.md)

**Pull request merged**